proposals/collection-expression-arguments.md

```diff
@@ -326,7 +326,8 @@ class MyBuilder
 ```
 
 If the target type is an *interface type*, then:
-* [*Overload resolution*](https://github.com/dotnet/csharpstandard/blob/standard-v7/standard/expressions.md#1264-overload-resolution) is used to determine the best instance constructor from the *argument list* from the following candidate signatures:
+* [*Overload resolution*](https://github.com/dotnet/csharpstandard/blob/standard-v7/standard/expressions.md#1264-overload-resolution) is used to determine the best candidate method signature.
+* The set of candidate signatures is the signatures below for the target interface that are applicable with respect to the *argument list* as defined in [*applicable function member*](https://github.com/dotnet/csharpstandard/blob/draft-v8/standard/expressions.md#12642-applicable-function-member). The signatures below use `new()` but are not necessarily constructors.
   * If the target type is `IEnumerable<E>`, `IReadOnlyCollection<E>`, or `IReadOnlyList<E>`, the candidates are:
     * `new()`
```

Comment:
i would remove this (and the other no-arg `new()`s) for now.  i think we should start minimal.  then see if LDM want to bring these in as part of a general conversation on no-arg withs(). :)